Core/src/org/sleuthkit/autopsy/commonfilesearch/CommonFilesPanel.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.sleuthkit.autopsy.commonfilesearch;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.Collections;
+import java.util.List;
+import java.util.logging.Level;
+import org.openide.util.Exceptions;
+import org.openide.util.NbBundle;
+import org.openide.windows.TopComponent;
+import org.sleuthkit.autopsy.casemodule.Case;
+import org.sleuthkit.autopsy.corecomponents.DataResultTopComponent;
+import org.sleuthkit.autopsy.corecomponents.TableFilterNode;
+import org.sleuthkit.autopsy.coreutils.Logger;
+import org.sleuthkit.datamodel.AbstractFile;
+import org.sleuthkit.datamodel.SleuthkitCase;
+import org.sleuthkit.datamodel.TskCoreException;
+
+/**
+ *
+ * @author bsweeney
+ */
+public class CommonFilesPanel extends javax.swing.JPanel {
+
+    /**
+     * Creates new form CommonFilesPanel
+     */
+    public CommonFilesPanel() {
+        initComponents();
+        customizeComponents();
+    }
+    
+    private void customizeComponents(){
+        addListenerToAll(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                search();
+            }
+        });
+    }
+    
+    void addListenerToAll(ActionListener l) {
+        this.searchButton.addActionListener(l);
+    }
+    
+    private void search(){
+        
+        String title = NbBundle.getMessage(this.getClass(), "CommonFilesPanel.search.results.title");
+        String pathText = NbBundle.getMessage(this.getClass(), "CommonFilesPanel.search.results.pathText");
+
+        //TODO somehow thats not deprecated???
```

Comment:
This method is definitely deprecated. Perhaps you need to merge in upstream/develop. The replacement method is Case.getOpenCase, which throws an exception.